.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--
+Thank you for reporting an issue. The more information you can give us, the
+better the chance we can fix your problem.
+
+This issue tracker is for bugs and issues found within node-gyp,
+if you have an issue installing a specific module, please file an issue on
+that module's issue tracker (find the link with `npm issues modulename`).
+
+Please fill in as much of the template below as you're able.
+
+Node Version: output of `node -v` and `npm -v`
+Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows)
+Compiler: output of `cc -v` (UNIX) or `msbuild.exe` (Windows)
+Module: the module you tried to install.
+-->
+
+* **Node Version**:
+* **Platform**:
+* **Compiler**:
+* **Module**:
+
+<details><summary>Verbose output (from npm or node-gyp):</summary>
+
+<!-- Paste your log between the backticks. If you can't find npm-debug.log then
+do `export npm_config_loglevel=silly` (UNIX) or `set npm_config_loglevel=silly`
+(Windows) before running the `npm install`/`node-gyp build` command, and paste
+the output.
+-->
+
+```
```

Comment:
Yeah, I only learned that by emailing Github support... Apparently it's a commonmark thing (it's in [their docs too](https://github.github.com/gfm/#html-blocks)).